action.yaml

```diff
@@ -341,6 +341,17 @@ runs:
         echo "OPTIONS=$options" >> $GITHUB_OUTPUT
       shell: bash
       id: deploy-options
+    - name: Determine if Development Mode is enabled
```

Comment:
I think you need to add `if [[ ${{ inputs.action }} == create-deployment-preview ]]; then` to both of these steps. The logic in these steps should only run if the action is create-deployment-preview

Reply:
No, we also would want to run this logic on `deploy-deployment-preview` or `deploy` action, I think.